client/src/components/Query/Filter/Specific.jsx

```diff
@@ -64,7 +64,9 @@ const optionObservedNeg = (externalDataSet) => (option) => {
 const sortOptions = (externalDataSet) => (options) => {
   const observedPos = options.filter(optionObservedPos(externalDataSet));
   const observedNeg = options.filter(optionObservedNeg(externalDataSet));
-  const notObserved = options.filter((o) => !optionObservedPos(externalDataSet)(o) && !optionObservedNeg(externalDataSet)(o));
+  const notObserved = options.filter((o) => (
```

Comment:
might not be worth it but just in case: I would put that result in a const `optionObservedPos(externalDataSet) `(same for `optionObservedNeg(externalDataSet)`) so that the computation is done only once.